plugins/spu2-x/src/Wavedump_wav.cpp

```diff
@@ -104,18 +104,25 @@ bool WavRecordEnabled = false;
 static WavOutFile *m_wavrecord = NULL;
 static Mutex WavRecordMutex;
 
-void RecordStart()
+void RecordStart(std::wstring* filename)
 {
     WavRecordEnabled = false;
 
     try {
         ScopedLock lock(WavRecordMutex);
         safe_delete(m_wavrecord);
-        m_wavrecord = new WavOutFile("recording.wav", 48000, 16, 2);
+#ifdef _WIN32
+		if (filename)
+			m_wavrecord = new WavOutFile((*filename) + "wav", 48000, 16, 2);
+		else
+			m_wavrecord = new WavOutFile("audio_recording.wav", 48000, 16, 2);
+#elif defined(__unix__)
+		m_wavrecord = new WavOutFile("audio_recording.wav", 48000, 16, 2);
```

Comment:
Here and the above chunk as well.

Reply:
I'm gonna have to probably do all that formatting on a separate PR... because oh my god, so much of the spu2-x code uses spaces instead of tabs. WHY???? I'll at least get these out of the way though.